src/games/strategy/triplea/delegate/MoveValidator.java

```diff
@@ -1675,17 +1699,18 @@ public static Route getBestRoute(final Territory start, final Territory end, fin
     if (!start.isWater() && !end.isWater()) {
       Route landRoute;
       if (isNeutralsImpassable) {
-        landRoute = data.getMap().getRoute_IgnoreEnd(start, end,
-            new CompositeMatchAnd<Territory>(Matches.TerritoryIsLand, noNeutral, noImpassible));
+        landRoute =
```

Comment:
It seems odd this was re-formatted. Please let us know if there is something actually off with the check-in formater, if not, please fix this so it is not formatted so weird. Maybe by assigning the CompositeMatchAnd to an intermediary variable, could get it formatted nicely that way perhaps.